src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaPlayerPresenter.cs

```diff
@@ -130,7 +130,10 @@ protected override Size MeasureOverride(Size availableSize)
 			if (double.IsNaN(Width) && double.IsNaN(Height))
 			{
 				availableSize.Width = availableSize.Width;
-				availableSize.Height = availableSize.Width / _currentRatio;
+				if (_currentRatio != 0)
```

Comment:
Good catch! You should apply it also for the `if (double.IsNaN(Height))` case :)